include/podio/RNTupleReader.h

```diff
@@ -1,24 +1,25 @@
 #ifndef PODIO_RNTUPLEREADER_H
 #define PODIO_RNTUPLEREADER_H
 
+#include "podio/GenericParameters.h"
 #include "podio/ROOTFrameData.h"
 #include "podio/SchemaEvolution.h"
 #include "podio/podioVersion.h"
 #include "podio/utilities/DatamodelRegistryIOHelpers.h"
 
+#include <ROOT/RVersion.hxx>
+
+#include <memory>
 #include <string>
 #include <string_view>
 #include <unordered_map>
 #include <vector>
-
-#include <ROOT/RNTuple.hxx>
-#include <ROOT/RNTupleModel.hxx>
-#include <RVersion.h>
 #if ROOT_VERSION_CODE >= ROOT_VERSION(6, 31, 0)
   #include <ROOT/RNTupleReader.hxx>
 #endif
```

Comment:
Since iwyu doesn't touch this, I would group it together with the include of `ROOT/RVersion.hxx` above.

Reply:
Done